### PR TITLE
Added page for Loops webhooks

### PR DESCRIPTION
**Pull request by danrowden**
This PR added a page for sending webhooks directly to [Loops](https://loops.so), a new feature just launched this week.

For reference: [Loops docs](https://loops.so/docs/integrations/clerk)

### CODE REVIEW

File: docs/integrations/webhooks/loops.mdx [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+6. Select the **Create** button.
+7. You will be redirected to your endpoint's settings page. Leave this page open.
+
+### Add your Signing secret in Loops
+
+To keep communication secure between the two platforms, you need to copy the signing secret created by Clerk for your endpoint.
+
+You can copy this by selecting the eye icon on your endpoint's settings page.
+
+Back in Loops, paste this signing secret in the **Signing Secret** field.
```
Comment (alexisintech) [inline docs/integrations/webhooks/loops.mdx:57]:
```suggestion
### Add your signing secret in Loops

To keep communication secure between the two platforms:

1. On the Clerk endpoint's settings page, copy the **Signing Secret**. It should be on the right side of the page with an eye icon next to it. 
1. Back in the Loops dashboard, add the **Signing Secret**.
```

**Review by alexisintech — APPROVED** [review]
Amazing addition - thank you so much for the work on this 😸💖

**Comment by danrowden** [comment]
Thanks @alexisintech! I noticed the step numbers aren't showing; it looks like my edit to make headings <h2> instead of <h3> removed them. I didn't realise they needed to be <h3> to have the step number. Shall I make another PR for this?

**Comment by danrowden** [comment]
New PR here: https://github.com/clerk/clerk-docs/pull/1254

**Pull request merged**